src/System.Security.Permissions/ref/System.Security.Permissions.cs

```diff
@@ -135,7 +156,54 @@ public sealed partial class OraclePermissionAttribute : System.Security.Permissi
         public bool ShouldSerializeKeyRestrictions() => false;
     }
 }
-
+namespace System.DirectoryServices
```

Comment:
Missing blank line?

Reply:
was intentional. should I bring it back?